app/views/components/_contents-list.html.erb

```diff
@@ -20,8 +20,12 @@
         <li
           class="app-c-contents-list__list-item app-c-contents-list__list-item--<%= parent_list_item_modifier %>"
         >
-          <%= link_to contents_item[:href], class: 'app-c-contents-list__link' do %>
-            <%= format_numbers ? wrap_numbers_with_spans(contents_item[:text]) : contents_item[:text] %>
+          <% if contents_item[:href] == nil %>
```

Comment:
There's probably no need to compare to nil here, you can use the truthiness of the return value of  `Hash#[]` to determine the condition.
What happens with numbers in the text only case? I see the links treat them differently. If the same treatment is needed it might be better to do this before deciding whether to render a link or not...

Reply:
Thanks for comments. Currently there isn't a case where you'd have numbers and no link, so I'd overlooked it, but yes, I'll go back and make sure that works as well (it likely doesn't right now).